integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -0,0 +1,510 @@
+
+        OAuth2Scope createdOAuth2Scope = createdAuthorizationServer.createOAuth2Scope(oAuth2Scope)
+
+        assertThat(createdOAuth2Scope, notNullValue())
+
+        OAuth2Scope tobeUpdatedOAuth2Scope = client.instantiate(OAuth2Scope)
+            .setName("java-sdk-authorization-server-scope" + RandomStringUtils.randomAlphanumeric(10) + "-updated")
+            .setConsent(OAuth2Scope.ConsentEnum.REQUIRED)
+            .setMetadataPublish(OAuth2Scope.MetadataPublishEnum.ALL_CLIENTS)
+
+        OAuth2Scope updatedOAuth2Scope = createdAuthorizationServer.updateOAuth2Scope(createdOAuth2Scope.getId(), tobeUpdatedOAuth2Scope)
+
+        assertThat(updatedOAuth2Scope.getId(), equalTo(tobeUpdatedOAuth2Scope.getId()))
+        assertThat(updatedOAuth2Scope.getName(), equalTo(tobeUpdatedOAuth2Scope.getName()))
+    }
+
+    @Test
+    void deleteOAuth2ScopesTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://example"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        OAuth2Scope oAuth2Scope = client.instantiate(OAuth2Scope)
+            .setName("java-sdk-authorization-server-scope" + RandomStringUtils.randomAlphanumeric(10))
+
+        OAuth2Scope createdOAuth2Scope = createdAuthorizationServer.createOAuth2Scope(oAuth2Scope)
+
+        assertThat(createdOAuth2Scope, notNullValue())
+
+        createdAuthorizationServer.deleteOAuth2Scope(createdOAuth2Scope.getId())
+
+        // delete may not effect immediately in the backend, so sleep
+        sleep(2000)
+
+        assertNotPresent(createdAuthorizationServer.listOAuth2Scopes(), createdOAuth2Scope)
+    }
+
+    // Claim operations
+
+    @Test
+    void listOAuth2ClaimsTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://example"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        OAuth2Claim oAuth2Claim = client.instantiate(OAuth2Claim)
+            .setName("java-sdk-authorization-server-claims" + RandomStringUtils.randomAlphanumeric(10))
+            .setStatus(OAuth2Claim.StatusEnum.INACTIVE)
+            .setClaimType(OAuth2Claim.ClaimTypeEnum.RESOURCE)
+            .setValueType(OAuth2Claim.ValueTypeEnum.EXPRESSION)
+            .setValue("\"driving!\"")
```

Comment:
It might be worth noting why the wrapped quotes here (and below),

Reply:
Sure, added a comment next to this line - `value must be an Okta EL expression if valueType is EXPRESSION`